tests/test_parser.py

```diff
@@ -77,6 +85,43 @@ def test_tls_defaults_from_config(self):
         self.assertEqual(args.cafile, 'foo.ca')
         self.assertEqual(args.port, 123)
 
+    @mock.patch('gvmtools.parser.Path')
+    def test_resolve_file_not_found_error(self, path_mock):
+        def resolve_raises_error():
+            raise FileNotFoundError
+
+        configpath = mock.MagicMock()
+        configpath.expanduser().resolve = mock.MagicMock(
+            side_effect=resolve_raises_error
+        )
+        path_mock.return_value = configpath
+
+        # pylint: disable=protected-access
+        return_value = self.parser._load_config('foobar')
```

Comment:
I am not a fan of testing internal behavior. The underscore in `_load_config` indicates that it should never be called from the outside and therefore is should *normally* also not called in tests directly. Could you try to modify your tests to call `parse_args` instead?